lms/djangoapps/instructor_task/models.py

```diff
@@ -270,6 +270,10 @@ def store_rows(self, course_id, filename, rows):
         strings), write the rows to the storage backend in csv format.
         """
         output_buffer = ContentFile('')
+
+        # Adding unicode signature (BOM) for MS Excel 2013 compatibility
+        output_buffer.write(u'\ufeff'.encode('utf8'))
```

Comment:
I think i would prefer that you use `output_buffer.write(codecs.BOM_UTF8)` here, and no need for blank lines.

Reply:
yes @nedbat, I prefer it too...